Framework/Core/include/Framework/InputRecord.h

```diff
@@ -136,7 +136,7 @@ class InputRecord {
 
   // If we ask for a char const *, we simply point to the payload. Notice this
   // is meant for C-style strings. If you want to actually get hold of the buffer,
-  // use get<DataRef> (or simply get) as that will give you the size as well.
+  // use get<DataRef*> (or simply get) as that will give you the size as well.
```

Comment:
I think this is a wrong catch

Reply:
indeed it is, luckily just a comment, fixed